src/styles/card.css

```
@@ -0,0 +1,84 @@
+.card {
+  width: 200px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 19px;
```

вот такие паддингы согласно макету

```suggestion
  padding: 32px 16px 16px;
```